### PR TITLE
Add documentation about serialize_with option

### PR DESCRIPTION
**Pull request by kragniz**
Maybe I didn't look hard enough, but I had to dig around in the code to
find you could do this. Maybe it would be helpful for the next person to
have that information close to the docs about betamax-serializers.

### CODE REVIEW

File: docs/third_party_packages.rst [hunk 1]
```diff
@@ -150,6 +150,14 @@ for you.
         session.request(method=method, url=url, ...)
 
 
+Instead of having to pass ``serialize_with='prettyjson'`` each time, we can
```
Comment (sigmavirus24) [inline docs/third_party_packages.rst]: So this breaks up the narrative voice in this section. If we can move this after the literal include that has the pretty-printed JSON, that would be great. I would also prefer this to say

```rst
If we want to use this serializer for every request, we can make this a global default by adding this to our configuration:

.. code-block:: python

    config.default_cassette_options['serialize_with'] = 'prettyjson'
```

**Comment by kragniz** [comment]
Sure, that was where I initially placed it. My problem with putting it down there was that since the json document above it is quite long, it would be easily missed.

**Comment by sigmavirus24** [comment]
What about moving it to https://betamax.readthedocs.org/en/latest/configuring.html?

**Comment by kragniz** [comment]
Even better idea!

File: docs/configuring.rst [hunk 2]
```diff
@@ -168,6 +168,16 @@ variables, e.g.,
 This means that you can run these tests on a service like Travis-CI without
 providing credentials.
 
+Setting default serializer
+``````````````````````````
+
+If you want to use a serializer for every request, you can add this to your
```
Comment (sigmavirus24) [inline docs/configuring.rst]: Alternate wording:

```rst
If you want to use a specific serializer for every cassette, you can set ``serialize_with`` as a default cassette option. For example, if you wanted to use the ``prettyjson`` serializer for every cassette you would do:

.. code-block:: python

    config.default_cassette_options['serialize_with'] = 'prettyjson'
```

**Comment by sigmavirus24** [comment]
:sparkles: :cake: :sparkles:

**Pull request merged**